src/main/java/com/alex/eshop/mapper/ItemCreateMapper.java

```diff
@@ -0,0 +1,12 @@
+package com.alex.eshop.mapper;
+
+import com.alex.eshop.dto.ItemCreateDTO;
+import com.alex.eshop.entity.Item;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+
+@Mapper
+public interface ItemCreateMapper {
```

Comment:
There is no reason to create Mapper for each DTO type. Use one mapper `ItemMapper` for all `ItemEntity-ItemAnyDto` mapping.